pegasus/sites.v3/code.org/public/yourschool.haml

```diff
@@ -5,11 +5,25 @@ social:
   "og:title": "Does your school teach computer science?"
   "og:description": "Expand computer science at your school or district. Join the thousands of schools who have already incorporated high quality computer science education into their curriculum and provide opportunities for the students in your local area."
 ---
+- require 'country_codes'
+- require 'state_abbr'
 %script{src: "https://cdn.optimizely.com/js/400912536.js"}
 - census_announcement = DCDO.get('census_announcement', nil)
 - if 'us' != hoc_detect_country
   - census_announcement ||= {}
   - census_announcement['hide_map'] = 'true'
+- if dashboard_user && dashboard_user[:school_info_id]
```

Comment:
If you get tired of seeing `-` you can also put all of this in a big `:ruby` block.